package.json

```diff
@@ -46,11 +46,13 @@
   },
 
   "scripts": {
-	"test": "mocha tests.coffee --compilers coffee:coffee-script/register"
+    "test": "mocha test/dice_test.coffee --compilers coffee:coffee-script/register"
```

Comment:
> Hey I just tested the command with just test/* and it worked.

:zap: shell globbing is the best and the worst thing ever





> Plurality makes more sense when there are many test files.

![picard](http://pixxx.wtf.cat/image/093d1v2g0C0R/628b747f8ccdfb757062f36a27eedecfc2295f515c0586e05fbfb0620c0571a2.jpg)